package.json

```diff
@@ -2,50 +2,82 @@
   "name": "css-loader",
   "version": "0.28.9",
   "author": "Tobias Koppers @sokra",
-  "license": "MIT",
   "description": "css loader module for webpack",
-  "engines": {
-    "node": ">=0.12.0 || >= 4.3.0 < 5.0.0 || >=5.10"
-  },
+  "license": "MIT",
+  "main": "dist/cjs.js",
   "files": [
-    "lib",
-    "index.js",
-    "locals.js"
+    "dist"
   ],
+  "scripts": {
+    "start": "npm run build -- -w",
+    "build": "cross-env NODE_ENV=production babel src -d dist --ignore 'src/**/*.test.js' --copy-files",
+    "clean": "del-cli dist",
+    "commitlint": "commitlint",
+    "commitmsg": "commitlint -e $GIT_PARAMS",
+    "lint": "eslint --cache src test",
+    "ci:lint:commits": "commitlint --from=${CIRCLE_BRANCH} --to=${CIRCLE_SHA1}",
+    "lint-staged": "lint-staged",
+    "prebuild": "npm run clean",
+    "prepublish": "npm run build",
+    "release": "standard-version",
+    "release:ci": "conventional-github-releaser -p angular",
+    "release:validate": "commitlint --from=$(git describe --tags --abbrev=0) --to=$(git rev-parse HEAD)",
+    "security": "nsp check",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:coverage": "jest --collectCoverageFrom='src/**/*.js' --coverage",
+    "ci:lint": "npm run lint && npm run security",
+    "ci:test": "npm run test -- --runInBand",
+    "ci:coverage": "npm run test:coverage -- --runInBand",
+    "defaults": "webpack-defaults"
+  },
   "dependencies": {
-    "babel-code-frame": "^6.26.0",
-    "css-selector-tokenizer": "^0.7.0",
-    "cssnano": "^3.10.0",
-    "icss-utils": "^2.1.0",
-    "loader-utils": "^1.0.2",
-    "lodash.camelcase": "^4.3.0",
-    "object-assign": "^4.1.1",
-    "postcss": "^5.0.6",
-    "postcss-modules-extract-imports": "^1.2.0",
-    "postcss-modules-local-by-default": "^1.2.0",
-    "postcss-modules-scope": "^1.1.0",
-    "postcss-modules-values": "^1.3.0",
+    "loader-utils": "^1.1.0",
+    "postcss": "^6.0.15",
```

Comment:
This line would solve https://github.com/webpack-contrib/css-loader/issues/691 in my case at least.